src/composables/useWalletList.ts

```
@@ -0,0 +1,77 @@
+  const initializedWallets = allRainbowWallets.map(wallet =>
+    wallet({ projectId, appName: 'MEW' }),
+  )
+
+  const newWalletList = computed<WalletConfig[]>(() => {
+    const newConArr: WalletConfig[] = []
+    initializedWallets.forEach(wallet => {
+      if (!DEFAULT_IDS.includes(wallet.id) && wallet.id !== 'ledger') {
+        const _types: WalletConfigType[] = []
+        if (
+          wallet.extension ||
+          (wallet.hasOwnProperty('installed') && !wallet.extension)
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)